src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -286,7 +286,7 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
             this.props.store.activeTranscript &&
             this.props.store.transcriptsByTranscriptId[this.props.store.activeTranscript] &&
             this.props.store.transcriptsByTranscriptId[this.props.store.activeTranscript].proteinLength) ||
-            Math.round(this.props.store.gene.length / 3);
+            0;
```

Comment:
can we set it to some non-zero value like 10?

Reply:
okay, use 10.

Reply:
replaced 0 with 10.